.github/workflows/release.yml

```
@@ -223,61 +381,67 @@ jobs:
       - name: Create release directory
+        if: steps.candidate_artifact.outputs.reused != 'true'
         run: mkdir -p release
 
       - name: Import Apple certificate
+        if: steps.candidate_artifact.outputs.reused != 'true'
         uses: apple-actions/import-codesign-certs@5142e029c445c10ffc7149d172e540235a065466 # v7
         with:
           p12-file-base64: ${{ secrets.APPLE_DEVELOPER_ID_CERT }}
           p12-password: ${{ secrets.APPLE_DEVELOPER_ID_PASSWORD }}
 
       - parallel:
```

actionlint (1.7.12)

[error] step must run script with "run" section or run action with "uses" section

(syntax-check)